frontend/app/src/components/profitloss/ReportMissingAcquisitions.vue

```diff
@@ -185,7 +185,7 @@ const childHeaders = computed<DataTableHeader[]>(() => {
 <style module lang="scss">
 .table {
   scroll-behavior: smooth;
-  max-height: calc(100vh - 310px);
+  max-height: calc(100vh - 340px);
   overflow: auto;
```

Comment:
this part looks like the others maybe it makes more sense o unify them than changing 4 different components